docs/en/operations/table_engines/index.md

```diff
@@ -2,15 +2,77 @@
 
 The table engine (type of table) determines:
 
-- How and where data is stored, where to write it to, and where to read it from.
+- How and where data is stored, where to write it to, and from where to read it.
```

Comment:
I think original version was more correct

Reply:
Let the editor judge us :-)